package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
You can refactor this file (just for practice and to keep things consistent) to reflect more on what your project is.
I'm gonna leave suggestions below!